lib/CXGN/Phenotypes/StorePhenotypes.pm

```diff
@@ -565,7 +565,8 @@ sub store {
                             $new_count++;
                         }
                         $check_unique_trait_stock{$trait_cvterm->cvterm_id(), $stock_id} = 1;
-                    } else {
+                    }
+                    if ($ignore_new_values) {
```

Comment:
Can this be somehow merged with the other same check that's happening on line 544?

Reply:
Oh yeah, I imagine I can just remove this one, good catch!

Reply:
Removed the duplicate code